### ISSUE TITLE
references stream_chat_flutter:android as the default plugin, but it does not provide an inline implementation. 3.27

### ISSUE THREAD

**Issue by erabti** [GetStream/stream-chat-flutter#2064]
### Which packages are you using?

stream_chat, stream_chat_flutter, stream_chat_localizations

### On what platforms did you experience the issue?

iOS, Android

### What version are you using?

8.3.0

### What happened?

After upgrading to latest flutter 3.27, a new warning is shown each time we run flutter pub get

Package stream_chat_flutter:android references stream_chat_flutter:android as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: android: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: android:` `pluginClass` or `dartPluginClass`.

### Steps to reproduce

```bash
run:
`flutter pub get`
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Ssiswent** [comment]
![image](https://github.com/user-attachments/assets/a8738d1a-5849-40ef-a861-3bc10b4c5f98)
The same.

**Comment by Abdullah-Arab** [comment]
same thing
```
Package stream_chat_flutter:android references stream_chat_flutter:android as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: android: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: android:` `pluginClass` or `dartPluginClass`.


Package stream_chat_localizations:android references stream_chat_localizations:android as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: android: default_package: stream_chat_localizations` or add
an inline implementation to stream_chat_localizations via `platforms: android:` `pluginClass` or `dartPluginClass`.


Package stream_chat_flutter:ios references stream_chat_flutter:ios as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: ios: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: ios:` `pluginClass` or `dartPluginClass`.


Package stream_chat_localizations:ios references stream_chat_localizations:ios as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: ios: default_package: stream_chat_localizations` or add an
inline implementation to stream_chat_localizations via `platforms: ios:` `pluginClass` or `dartPluginClass`.


Package stream_chat_flutter:linux references stream_chat_flutter:linux as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: linux: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: linux:` `pluginClass` or `dartPluginClass`.


Package stream_chat_localizations:linux references stream_chat_localizations:linux as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: linux: default_package: stream_chat_localizations` or add an
inline implementation to stream_chat_localizations via `platforms: linux:` `pluginClass` or `dartPluginClass`.


Package stream_chat_flutter:macos references stream_chat_flutter:macos as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: macos: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: macos:` `pluginClass` or `dartPluginClass`.


Package stream_chat_localizations:macos references stream_chat_localizations:macos as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: macos: default_package: stream_chat_localizations` or add an
inline implementation to stream_chat_localizations via `platforms: macos:` `pluginClass` or `dartPluginClass`.


Package stream_chat_flutter:windows references stream_chat_flutter:windows as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: windows: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: windows:` `pluginClass` or `dartPluginClass`.


Package stream_chat_localizations:windows references stream_chat_localizations:windows as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: windows: default_package: stream_chat_localizations` or add
an inline implementation to stream_chat_localizations via `platforms: windows:` `pluginClass` or `dartPluginClass`.


Package stream_chat_flutter:android references stream_chat_flutter:android as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: android: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: android:` `pluginClass` or `dartPluginClass`.


Package stream_chat_localizations:android references stream_chat_localizations:android as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: android: default_package: stream_chat_localizations` or add
an inline implementation to stream_chat_localizations via `platforms: android:` `pluginClass` or `dartPluginClass`.


Package stream_chat_flutter:ios references stream_chat_flutter:ios as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: ios: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: ios:` `pluginClass` or `dartPluginClass`.


Package stream_chat_localizations:ios references stream_chat_localizations:ios as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: ios: default_package: stream_chat_localizations` or add an
inline implementation to stream_chat_localizations via `platforms: ios:` `pluginClass` or `dartPluginClass`.


Package stream_chat_flutter:linux references stream_chat_flutter:linux as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: linux: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: linux:` `pluginClass` or `dartPluginClass`.


Package stream_chat_localizations:linux references stream_chat_localizations:linux as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: linux: default_package: stream_chat_localizations` or add an
inline implementation to stream_chat_localizations via `platforms: linux:` `pluginClass` or `dartPluginClass`.


Package stream_chat_flutter:macos references stream_chat_flutter:macos as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: macos: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: macos:` `pluginClass` or `dartPluginClass`.


Package stream_chat_localizations:macos references stream_chat_localizations:macos as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: macos: default_package: stream_chat_localizations` or add an
inline implementation to stream_chat_localizations via `platforms: macos:` `pluginClass` or `dartPluginClass`.


Package stream_chat_flutter:windows references stream_chat_flutter:windows as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: windows: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: windows:` `pluginClass` or `dartPluginClass`.


Package stream_chat_localizations:windows references stream_chat_localizations:windows as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: windows: default_package: stream_chat_localizations` or add
an inline implementation to stream_chat_localizations via `platforms: windows:` `pluginClass` or `dartPluginClass`.


Package stream_chat_flutter:android references stream_chat_flutter:android as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: android: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: android:` `pluginClass` or `dartPluginClass`.


Package stream_chat_localizations:android references stream_chat_localizations:android as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: android: default_package: stream_chat_localizations` or add
an inline implementation to stream_chat_localizations via `platforms: android:` `pluginClass` or `dartPluginClass`.


Package stream_chat_flutter:ios references stream_chat_flutter:ios as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: ios: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: ios:` `pluginClass` or `dartPluginClass`.


Package stream_chat_localizations:ios references stream_chat_localizations:ios as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: ios: default_package: stream_chat_localizations` or add an
inline implementation to stream_chat_localizations via `platforms: ios:` `pluginClass` or `dartPluginClass`.


Package stream_chat_flutter:linux references stream_chat_flutter:linux as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: linux: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: linux:` `pluginClass` or `dartPluginClass`.


Package stream_chat_localizations:linux references stream_chat_localizations:linux as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: linux: default_package: stream_chat_localizations` or add an
inline implementation to stream_chat_localizations via `platforms: linux:` `pluginClass` or `dartPluginClass`.


Package stream_chat_flutter:macos references stream_chat_flutter:macos as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: macos: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: macos:` `pluginClass` or `dartPluginClass`.


Package stream_chat_localizations:macos references stream_chat_localizations:macos as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: macos: default_package: stream_chat_localizations` or add an
inline implementation to stream_chat_localizations via `platforms: macos:` `pluginClass` or `dartPluginClass`.


Package stream_chat_flutter:windows references stream_chat_flutter:windows as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: windows: default_package: stream_chat_flutter` or add an inline
implementation to stream_chat_flutter via `platforms: windows:` `pluginClass` or `dartPluginClass`.


Package stream_chat_localizations:windows references stream_chat_localizations:windows as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: windows: default_package: stream_chat_localizations` or add
an inline implementation to stream_chat_localizations via `platforms: windows:` `pluginClass` or `dartPluginClass`.



Package stream_chat_flutter:android references stream_chat_flutter:android as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: android: default_package: stream_chat_flutter` or add an inline implementation to stream_chat_flutter via `platforms: android:` `pluginClass` or `dartPluginClass`.

Package stream_chat_localizations:android references stream_chat_localizations:android as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: android: default_package: stream_chat_localizations` or add an inline implementation to stream_chat_localizations via `platforms: android:` `pluginClass` or `dartPluginClass`.

Package stream_chat_flutter:ios references stream_chat_flutter:ios as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: ios: default_package: stream_chat_flutter` or add an inline implementation to stream_chat_flutter via `platforms: ios:` `pluginClass` or `dartPluginClass`.

Package stream_chat_localizations:ios references stream_chat_localizations:ios as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: ios: default_package: stream_chat_localizations` or add an inline implementation to stream_chat_localizations via `platforms: ios:` `pluginClass` or `dartPluginClass`.

Package stream_chat_flutter:linux references stream_chat_flutter:linux as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: linux: default_package: stream_chat_flutter` or add an inline implementation to stream_chat_flutter via `platforms: linux:` `pluginClass` or `dartPluginClass`.

Package stream_chat_localizations:linux references stream_chat_localizations:linux as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: linux: default_package: stream_chat_localizations` or add an inline implementation to stream_chat_localizations via `platforms: linux:` `pluginClass` or `dartPluginClass`.

Package stream_chat_flutter:macos references stream_chat_flutter:macos as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: macos: default_package: stream_chat_flutter` or add an inline implementation to stream_chat_flutter via `platforms: macos:` `pluginClass` or `dartPluginClass`.

Package stream_chat_localizations:macos references stream_chat_localizations:macos as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: macos: default_package: stream_chat_localizations` or add an inline implementation to stream_chat_localizations via `platforms: macos:` `pluginClass` or `dartPluginClass`.

Package stream_chat_flutter:windows references stream_chat_flutter:windows as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_flutter to either avoid referencing a default implementation via `platforms: windows: default_package: stream_chat_flutter` or add an inline implementation to stream_chat_flutter via `platforms: windows:` `pluginClass` or `dartPluginClass`.

Package stream_chat_localizations:windows references stream_chat_localizations:windows as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_chat_localizations to either avoid referencing a default implementation via `platforms: windows: default_package: stream_chat_localizations` or add an inline implementation to stream_chat_localizations via `platforms: windows:` `pluginClass` or `dartPluginClass`.

lib/ui/l10n/stream_chat_localization_ar.dart:24:7: Error: The non-abstract class 'StreamChatLocalizationsAr' is missing implementations for these members:
 - Translations.editedMessageLabel
Try to either
 - provide an implementation,
 - inherit an implementation from a superclass or mixin,
 - mark the class as abstract, or
 - provide a 'noSuchMethod' implementation.

class StreamChatLocalizationsAr extends GlobalStreamChatLocalizations {
      ^^^^^^^^^^^^^^^^^^^^^^^^^
../../../.pub-cache/hosted/pub.dev/stream_chat_flutter-8.3.0/lib/src/localization/translations.dart:73:14: Context: 'Translations.editedMessageLabel' is defined here.
  String get editedMessageLabel;
             ^^^^^^^^^^^^^^^^^^
Target kernel_snapshot_program failed: Exception


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileFlutterBuildRelease'.
> Process 'command '/Users/thedev/Development/flutter/bin/flutter'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 48s
Running Gradle task 'assembleRelease'...                           49.5s
Gradle task assembleRelease failed with exit code 
```

**Comment by xsahil03x** [comment]
Should be fixed in #2068
